server/network/aoprotocol.py

```diff
@@ -482,6 +482,118 @@ def net_cmd_ms(self, args):
                 self.client.send_ooc('You don\'t any areas!')
                 return
             text = ' '.join(part[1:])
+        elif text.startswith('/testify '): # Start a new testimony in this area.
```

Comment:
Yikes. IC commands. Maybe extract part of these command handlers to area functions.

Reply:
Done, 7361ecd